syntax = "proto3";
 
package transport_catalogue_protobuf;
 
message RoutingSettings {
    double bus_wait_time = 1;
    double bus_velocity = 2;
}